package.json

```diff
@@ -4,9 +4,11 @@
   "description": "",
   "repository": "https://github.com/Spiritdude/OpenJSCAD.org",
   "scripts": {
-    "test": "npm i && npm run bootstrap && lerna run test",
+    "test-one": "lerna run --stream --scope '@jscad/json-deserializer' test",
```

Comment:
what is the purpose of this one ?
is this a temporary thing ?

Reply:
I was trying to find a simple way to initiate tests for a single module. This would be very handy for making bug fixes to any of the modules.

Of course, the tests can be initiated from the specific directory. But this feels more ‘built-in’.

Not sure as to which is the best approach.

Reply:
initialise tests ? as in running all the boilerplate / setup code ?

Reply:
removed